src/components/Match/MatchGraph.jsx

```diff
@@ -49,9 +49,12 @@ const drawGraphs = (props, id) => {
       },
       tooltip: {
         contents(d, defaultTitleFormat, defaultValueFormat, color) {
-          d.sort((a, b) => b.value - a.value);
           return this.getTooltipContent(d, defaultTitleFormat, valueFormat || defaultValueFormat, color);
         },
+        order: (a, b) => {
+          if (a.id === 'Gold' || b.id === 'Gold') return a.id - b.id;
```

Comment:
Can you check if `props.type === 'difference'` here to determine whether to sort?